src/utils/openCloseComponent.ts

```diff
@@ -8,20 +8,50 @@ export interface OpenCloseComponent {
    */
   readonly el: HTMLElement;
 
+  /**
+   * When true, the component opens.
+   */
+  open?: boolean;
```

Comment:
We'll need this to not be optional before we merge. 👍

Reply:
Ok, I think `alert` is the component I'm waiting for to deprecate the `active` prop in favor of `open`. And then I'll make `open` mandatory.